code/modules/transport/pods/vehicle.dm

```diff
@@ -578,6 +581,12 @@
 				SPAWN_DBG(0)
 					M.throw_at(throw_at, movement_controller:velocity_magnitude, 2)
 				logTheThing("combat", src, target, "crashes into [target] [log_loc(src)].")
+				SPAWN_DBG(2.5 SECONDS)
+					if(M.health < 0) 
+						vehicular_manslaughter = 0 //we now check if person was sent into crit after hit, if they did we get the achievement
```

Comment:
This does the opposite of what you want (gives the medal only when you *don't* put the person into crit).